xml_converter/attribute_test_script.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+ATTRIBUTE_TEST_CASES_FOLDER_PATHS=("./test_cases/can_fade_tests")
+
+ATTRIBUTE_TEST_CASES=()
+
+for ATTRIBUTE_TEST_CASES_FOLDER_PATH in "${ATTRIBUTE_TEST_CASES_FOLDER_PATHS[@]}"; do
```

Comment:
Not a hard rule, but using SCREAMING_SNAKE_CASE is sometimes reserved for constants or globals or "something other then a normal variable". in this case `ATTRIBUTE_TEST_CASES_FOLDER_PATH` is a normal variable.